pkg/costattribution/active_tracker.go

```diff
@@ -16,10 +16,18 @@ import (
 	"go.uber.org/atomic"
 )
 
+type asCounters struct {
+	asCount     *atomic.Int64
+	nhCount     *atomic.Int64
+	nhBucketNum *atomic.Int64
+}
```

Comment:
Is there any reason why you chose to have struct of pointers instead of a pointer to a struct with just atomic.Int64 values?